stdlib/docs/development.md

```diff
@@ -112,15 +112,23 @@ Otherwise, CI will fail in its lint and formatting checks.  The `mojo` compiler
 provides a `format` command.  So, you can format your changes like so:
 
 ```bash
-mojo format -l 80 <file1> <file2> ...
+mojo format -l 80 ./
```

Comment:
**Suggestion** Remove the parts about `-l 80` - see https://github.com/modularml/mojo/pull/2244. Here and elsewhere in this file when you rebase.